runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -96,6 +96,9 @@ public Dimension render(Graphics2D graphics)
 		// Tooltip action type handling
 		switch (action)
 		{
+			case ITEM_USE_ON_WIDGET:
+				if (!menuEntry.getTarget().contains("High Level Alchemy") ||
```

Comment:
By not using single line ifs I did not meant splitting single line if to 2 lines, I meant use { } like we do everywhere else